.eslintrc.js

```diff
@@ -1,7 +1,9 @@
 module.exports = {
-    "env": {
-        "es6": true,
-        "browser": true
-    },
-    "extends": "eslint-config-digitalbazaar",
+  env: {
+    es6: true,
```

Comment:
This is already in: https://github.com/digitalbazaar/eslint-config-digitalbazaar/blob/master/index.js#L3

Just trying to avoid extra boilerplate getting copied all over the place.